### PR TITLE
Ran - Scoring to Amp or Shooter Automatically and clean code

### PR DESCRIPTION
**Pull request by bossyshrimp68**

### CODE REVIEW

**Review by greenblitz4590 — APPROVED** [review]

File: src/main/java/edu/greenblitz/robotName/commands/shooter/shootingState/ShootAutomatically.java [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package edu.greenblitz.robotName.commands.shooter.shootingState;
+
+import edu.greenblitz.robotName.shootingStateService.ShootingPositionConstants;
+
+public class ShootAutomatically extends GoToShootingStateAndShoot{
```
Comment (YoniKiriaty) [inline src/main/java/edu/greenblitz/robotName/commands/shooter/shootingState/ShootAutomatically.java:5]: ShootAutomatically is a misleading name.

File: src/main/java/edu/greenblitz/robotName/commands/ShootOrScoreToAmp.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+import edu.greenblitz.robotName.ScoringModeSelector;
+import edu.greenblitz.robotName.commands.arm.ScoreToAmp;
+import edu.greenblitz.robotName.commands.shooter.shootingState.GoToShootingStateAndShoot;
+import edu.greenblitz.robotName.commands.shooter.shootingState.ShootAutomatically;
+import edu.greenblitz.robotName.shootingStateService.ShootingPositionConstants;
+import edu.greenblitz.robotName.shootingStateService.ShootingStateCalculations;
+import edu.wpi.first.wpilibj2.command.ConditionalCommand;
+
+public class ShootOrScoreToAmp extends ConditionalCommand {
```
Comment (YoniKiriaty) [inline src/main/java/edu/greenblitz/robotName/commands/ShootOrScoreToAmp.java:11]: maybe
```suggestion
public class ScoreToCurrentScoringMode extends ConditionalCommand {
```
or something idk

File: src/main/java/edu/greenblitz/robotName/commands/shooter/shootingState/ShootAutomatically.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    public ShootAutomatically(){
```
Comment (YoniKiriaty) [inline src/main/java/edu/greenblitz/robotName/commands/shooter/shootingState/ShootAutomatically.java:6]: space

File: src/main/java/edu/greenblitz/robotName/commands/ShootOrScoreToAmp.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    public ShootOrScoreToAmp(){
```
Comment (YoniKiriaty) [inline src/main/java/edu/greenblitz/robotName/commands/ShootOrScoreToAmp.java:12]: space

**Review by YoniKiriaty — COMMENTED** [review]
i like spaces

**Review by yonnbro — APPROVED** [review]
After Yoni’s comments

**Review by YoniKiriaty — APPROVED** [review]

**Review by greenblitz4590 — APPROVED** [review]
Yoni - approved

**Pull request merged**